src/main/java/io/github/noeppi_noeppi/libx/mod/ModXRegistration.java

```diff
@@ -17,14 +23,52 @@
 import java.util.function.Consumer;
 import java.util.function.Predicate;
 
+/**
+ * You should extends this instead of {@link ModX} if you want to use the LibX registration system.
```

Comment:
Maybe `You need to` instead of `You should` because you need it to use the LibX registration system instead of "it would be good to extend it"